src/MSBuild/Resources/Strings.resx

```diff
@@ -1797,12 +1797,11 @@
   <data name="SAC_Off" xml:space="preserve">
     <value>0: turned off</value>
   </data>
-  <!-- **** TerminalLogger strings end **** -->
-    <!--
-        The command line message bucket is: MSB1001 - MSB1999
-
-        Next error code should be MSB1069.
-
-        Don't forget to update this comment after using the new code.
-  -->
-</root>
+  <data name="HelpMessage_52_BuildCheckSwitch" xml:space="preserve">
+    <value>  -check
+                     Enables BuildChecks during the build. BuildCheck is a series
+                     of checks to ensure code quality for build via rule violation reports.</value>
+    <comment>      LOCALIZATION: "-check", and "-BuildChecks" should not be localized.
```

Comment:
There's a loc comment you can use to force the loc tooling to leave these strings:

```
<comment>{Locked="-check"}{Locked="BuildChecks"}{MaxLength=80}</comment>
```

You can put the rest of the context in the comment, but these directives should be supported by the loc tooling. I'll DM you an internal link to the docs for this.